### ISSUE TITLE
Tutorial 8: Networks - faulty link to Keras' network.py

### ISSUE THREAD

**Issue by jan-gebauer** [tensorflow/agents#555]
Hello,

in Tutorial 8: Networks, there's a faulty link to Keras' network.py in the section Defining Networks. I am not sure how this bug came about but my guess is that Keras removed this API/file/whatever, reworked it into Layers API, which was then propagated into TF-agents and a new network.py was made here.

I am not sure what the best overall fix is as I don't know who the target audience of the tutorial is but here's what would help me:

> In TF-Agents we **use Network class (link to TF-Agents' network.py), which is a modification of Keras' Layers API (link to the API).** With it we can:

Change is in bold. I've shopped around for a RL framework, so I am somewhat familiar with Keras and learning that the Network class inherits from Layers without going into the code would make it easier for me to make a mental model when reading the tutorial for the first time.

**Comment by ebrevdo** [comment]
Yes; this is outdated.  Thanks for catching!  Keras used to have a semi-public Network class but got rid of it; and we no longer use it.  @tfboyd any cycles to remove this comment and link to Keras' Network class?
